spec/factories/auctions.rb

```diff
@@ -116,6 +116,7 @@
     end
 
     trait :accepted do
+      ended_at { Time.now - 1.days }
```

Comment:
For seed data to be displayed in /insights, I thought the auction needs to have ended to be counted. But `result: :accepted` might be sufficient here.